openedx/core/lib/api/tests/test_authentication.py

```diff
@@ -74,11 +79,13 @@ class OAuth2AuthenticationDebug(authentication.OAuth2AuthenticationAllowInactive
 @ddt.ddt
 @unittest.skipUnless(settings.FEATURES.get("ENABLE_OAUTH2_PROVIDER"), "OAuth2 not enabled")
 @override_settings(ROOT_URLCONF=__name__)
-class OAuth2Tests(TestCase):
-    """OAuth 2.0 authentication"""
+class OAuth2DeprecatedTests(TestCase):
+    """Tets for OAuth2AllowInActiveUsers Deprecated class"""
```

Comment:
1. Typo: Tets
2. Rename to `OAuth2AllowInActiveUsersTests` and kill the comment.